src/Home/Home.tsx

```diff
@@ -377,7 +377,9 @@ const style = StyleSheet.create({
     ...Typography.header4,
   },
   shareIconContainer: {
-    paddingHorizontal: Spacing.medium,
+    width: 60,
```

Comment:
Thoughts on this living in a variable? Something like `const activationStatusRightWidth = 60` right above the stylesheet?

Reply:
Hmm, yeah I can see that -- I'll make the change